.github/workflows/tests-all.yml

```diff
@@ -35,11 +34,16 @@ jobs:
         pip install --upgrade -r tests/requirements.txt
         pip install tox awscli
         python -m awscli configure set region us-east-1
-    - name: Text examples
+    - name: Run all the tests
       run: |
-        python3 -m tox -- -v -m "examples"
+        tox -- -v
```

Comment:
This will not allow parallelizing these tests. And it may not be safe for coverage.